moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Box, Grid, Typography } from "@mui/material";
+
+/**
+ * ProjectSummaryCouncilDistricts Component
+ * @param {Object} data - The project geography from the graphql query's data object
+ * @param {Object} classes - The shared style settings
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ProjectSummaryCouncilDistricts = ({ projectGeography, classes }) => {
+  // reduce the array of geography objects into an array of city council districts
+  const getDistricts = (data) => {
+    const initialValue = [];
+    const districts = data.reduce(
+      (acc, component) => [...acc, component["council_districts"]],
+      initialValue
+    );
+
+    return [...new Set(districts.flat())];
+  };
+
+  return (
+    <Grid item xs={12} className={classes.fieldGridItem}>
+      <Typography className={classes.fieldLabel}>
+        Council district(s):
+      </Typography>
+      <Box className={classes.fieldBox}>
+        <Typography className={classes.fieldLabelTextNoHover}>
+          {getDistricts(projectGeography).join(", ")}
```

Comment:
Is the order the districts shown important? What about printing them in ascending order, like maybe sticking `.sort((a, b) => a - b)` in there. Totally no big deal though!

Reply:
i proposed the same question in my PR description, im not sure! i kinda want it ordered, but I didnt know if it was just me

Reply:
Oh, 🤦, I so totally missed that! You are way ahead of me. I'm with you, numeric order seems like a good default.

Reply:
yes, thanks for flagging this. please order them A-Z!